module/VuFind/src/VuFind/Autocomplete/SolrPrefix.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * Solr Autocomplete Module
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2021.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
```

Comment:
This is an outdated address; this line should be:

`* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA`